bot.py

```diff
@@ -1232,6 +1235,22 @@ def check(answer):
     exit(0)
     return
 
+@bot.command(aliases=["image"])
```

Comment:
Probably want a more descriptive function (used for the command) name and not need an alias.